Cpix/DrmSystem.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.IO;
+using System.Xml;
+using System.Xml.XPath;
+
+namespace Axinom.Cpix
+{
+	public sealed class DrmSystem : Entity
+	{
+		/// <summary>
+		/// Gets or sets the unique ID of the DRM system. Refer to the DASH-IF System
+		/// ID registry for a list of DRM system IDs.
+		/// </summary>
+		public Guid Id { get; set; }
+
+		/// <summary>
+		/// Gets or sets the ID of the content key the DRM system references.
+		/// </summary>
+		public Guid KeyId { get; set; }
+
+		/// <summary>
+		/// Gets or sets the content protection data, which is the full well-formed
+		/// XML fragment that shall be added under the ContentProtection element in a
+		/// DASH manifest. This must be a UTF-8 XML string without a byte order mark,
+		/// encoded as base64.
+		/// </summary>
+		public string ContentProtectionData { get; set; }
```

Comment:
Our library should make it as easy as possible for the user, so let's do all the work we can for them. For example, we should take care of any base64 encoding/decoding ourselves. This would allow us to expose the inner string directly, without leaving encoding as a concern for the user.

Reply:
Yes.